src/powerbox/import_fft.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
+import warnings
+
+try:
+    import pyfftw
+except ImportError:
+    pass
+
+
+class FFTBackend:
+    def ifftn(self):
+        pass
+
+    def fftn(self):
+        pass
+
+    def fftshift(self):
```

Comment:
You can actually make the code a *little* simpler by defining `fftshift` and `ifftshift` directly in the base class here. Just let them refer to `self._fftshift`, which should be `np.fft.fftshift` on the numpy and vice versa on pyfftw.